src/components/Toast/types.tsx

```diff
@@ -0,0 +1,24 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ReactChild } from 'react';
+
+export type ToastProps = {
+    isOpen: boolean;
+    style: ToastStyles;
+    icon: ReactChild;
+    children?: ReactChild;
```

Comment:
```suggestion
    icon: ReactNode;
    children?: ReactNode;
```

`ReactChild` is deprecated in React 18